CHANGELOG.md

```diff
@@ -1,3 +1,160 @@
+# 85.2.0
+
+In this release we have added new features to Banner and Tabs, fixed bugs in Buttons and FlexibleHeader, and improved test coverage in BottomSheet. Additionally, we have added a TextField extension that will eventually house an MDCTextField alternative.
+
+## New features
+
+MDCBanner now supports `intrinsicContentSize`.
+
+```objc
+MDCBannerView *myBanner = ...
+CGSize validIntrinsicContentSize = myBanner.intrinsicContentSize;
+```
+
+MDCTabBarView has several new core features implemented.
+
+```objc
+MDCTabBarView *myTabBarView = ...
+myTabBarView.delegate = self;
+[myTabBarView setTitleFont:myFont forState:UIControlStateNormal];
+[myTabBarView setTitleColor:myColor forState:UIControlStateNormal];
```

Comment:
I can't tell what this code snippet is demonstrating; are these all new APIs?

Reply:
This is a sampling of some of the many new APIs on the new Tabs. I should make that more clear!

Reply:
I got rid of the snippet.